src/scenes/AllEvents/index.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import appActions from '../../redux/App/actions';
+import graphqlActions from '../../redux/Graphql/actions';
+import InfiniteScroll from '../../components/InfiniteScroll';
+import theme from '../../config/theme';
+import EventCard from '../../components/EventCard';
+import { Token, OracleStatus } from '../../constants';
+import TopActions from '../Dashboard/components/TopActions';
+
+const LIMIT = 6;
```

Comment:
Henry found that if the limit was too low, there was an issue with the pagination. That's why we set it as limit = 50.

Reply:
oh shoot. Yeah I meant to increase that back.